sorc/checkout.sh

```diff
@@ -1,110 +1,87 @@
 #!/bin/sh
-set -xu
+set -xue
 
 topdir=$(pwd)
 echo $topdir
 
-if [ $# -eq 1 ]; then
-model=$1
-fi
-model=${model:-"uncoupled"}
-
-if [ $model = "coupled" ]; then
-echo fv3_mom6_cice5 checkout ...
-rm -f ${topdir}/checkout-fv3_coupled.log
-
-if [[ ! -d fv3_coupled.fd ]] ; then
-    git clone https://github.com/ufs-community/ufs-s2s-model fv3_coupled.fd >> ${topdir}/checkout-fv3_coupled.log 2>&1
```

Comment:
I think we still want to checkout the coupled model?   Or did we move too externals with this commit as well?

Reply:
Oh. This is wrong...